linker.go

```diff
@@ -259,6 +260,8 @@ func fixupAndValidate(insns asm.Instructions) error {
 	return nil
 }
 
+const kfuncCallPoisonBase = 2002000000
```

Comment:
Can you add a link with the source of the magic number? Guessing that this is from libbpf.